srcpkgs/wildmidi/template

```diff
@@ -13,7 +13,7 @@ maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
 license="GPL-3.0-or-later, LGPL-3.0-only"
 homepage="http://www.mindwerks.net/projects/wildmidi"
```

Comment:
Would you mind switching to `https` here?

Reply:
Sure.